src/blog/wasm-decompile.md

```diff
@@ -0,0 +1,225 @@
+---
+title: "What's in that .wasm? Introducing: wasm-decompile"
+author: 'Wouter van Oortmerssen'
+avatars:
+  - 'wouter-van-oortmerssen'
+date: 2020-04-20 12:35:00
+tags:
+  - WebAssembly
+  - tooling
+description: 'WABT gains a new decompilation tool that can make it easier to
+read the contents of Wasm modules'
+---
+We have a growing number of compilers and other tools that generate or
+manipulate `.wasm` files, and sometimes you might want to have a look inside.
+Maybe you're a developer of such a tool, or more directly, you're simply a
+programmer targeting Wasm, and wondering what the generated code looks like,
+for performance or other reasons.
+
+Problem is, Wasm is rather low level, much like actual assembly code. In particular,
+unlike say, the JVM, all data structures have been compiled down to load/store
+operations, rather than conveniently named classes and fields. Compilers like
+LLVM can do an impressive amount of transformations that make the generated code
+look nothing like the code that went in.
+
+## Disassemble or.. decompile?
+
+You could use tools like `wasm2wat` (part of the
+[WABT](https://github.com/WebAssembly/wabt) toolkit), to transform a `.wasm`
+into Wasm's standard text format, `.wat`, which is a very faithful but not
+particularly readable representation.
+
+For example, a simple C function like a dot product:
+
+```C
```

Comment:
Looks like CI failed because of this: https://travis-ci.com/github/v8/v8.dev/jobs/321362009#L341

I think you need to use lowercase tag (`c`) or maybe even `cpp` in case that one doesn't work.